README.md

```diff
@@ -4,130 +4,182 @@
 
 # Swift for TensorFlow
 
-Welcome to the Swift for TensorFlow development community! For discussions, join the [swift@tensorflow.org mailing list](https://groups.google.com/a/tensorflow.org/d/forum/swift).
+> Swift for TensorFlow: No barriers.
+
+Swift for TensorFlow (aka S4TF) is a next-generation platform for machine
```

Comment:
I think this "S4TF" should be removed as well for more clear, consistent branding.

Reply:
Done!